src/keepass.cpp

```diff
@@ -341,7 +341,6 @@ void CKeePassIntegrator::doHTTPPost(const std::string& sRequest, int& nStatusRet
     evhttp_add_header(output_headers, "Connection", "close");
 
     // Logging of actual post data disabled as to not write passphrase in debug.log. Only enable temporarily when needed
```

Comment:
Maybe also remove this comment as it was referring to the commented out log print